index.js

```diff
@@ -0,0 +1,618 @@
+            changeLangForRissia(document.querySelector('.row-4').children, russianKeybord[3]);
+            changeLangForRissia(document.querySelector('.row-5').children, russianKeybord[4]);
+        } else if (lg == 'russia') {
+            changeLangForEnglish(document.querySelector('.row-1').children, englishKeybord[0]);
+            changeLangForEnglish(document.querySelector('.row-2').children, englishKeybord[1]);
+            changeLangForEnglish(document.querySelector('.row-3').children, englishKeybord[2]);
+            changeLangForEnglish(document.querySelector('.row-4').children, englishKeybord[3]);
+            changeLangForEnglish(document.querySelector('.row-5').children, englishKeybord[4]);
+        }
+    }
+    localStorage.setItem('keyOfLang', lg);
+});
+
+
+///////////////////////////Created function for selected size
+let size = 'small';
+
+function changeSizeforLarge(arr, array, j) {
+    let i = 0;
+    for (let elem of arr) {
+        if (i <= j) {
+
+            if (array[i][0] == 'ShiftRight' || array[i][0] == 'ShiftLeft') {
+                elem.innerHTML = 'Shift';
+            } else if (array[i][0] == 'ControlRight' || array[i][0] == 'ControlLeft' || array[i][1] == 'ControlLeft') {
+                elem.innerHTML = 'Control';
+            } else if (array[i][0] == 'AltRight' || array[i][0] == 'AltLeft') {
+                elem.innerHTML = 'Alt';
+            } else if (array[i][0] == 'Backquote') {
+                elem.innerHTML = '~';
+            } else if (array[i][0] == 'MetaLeft') {
+                elem.innerHTML = 'Win';
+            } else {
+                elem.innerHTML = array[i][1];
+            }
+        }
+        i++;
+    }
+    size = 'large';
+}
+
+function changeSizeforSmall(arr, array, j) {
+    let i = 0;
+    for (let elem of arr) {
+        if (i <= j) {
+            if (array[i][0] == 'ShiftRight' || array[i][0] == 'ShiftLeft') {
+                elem.innerHTML = 'Shift';
+            } else if (array[i][0] == 'ControlRight' || array[i][0] == 'ControlLeft' || array[i][1] == 'ControlLeft') {
+                elem.innerHTML = 'Control';
+            } else if (array[i][0] == 'AltRight' || array[i][0] == 'AltLeft') {
+                elem.innerHTML = 'Alt';
+            } else if (array[i][0] == 'Backquote') {
+                elem.innerHTML = '~';
+            } else if (array[i][0] == 'MetaLeft') {
+                elem.innerHTML = 'Win';
+            } else {
+                elem.innerHTML = array[i][0];
+            }
+        }
+
+        i++;
+    }
+    size = 'small';
+}
+
+
+///////////////////////////Created event click at the button
+keybourd.addEventListener('click', function(event) {
+    let button = event.target.closest('button');
+
+    if (!button) {
+        return
+    }
+
+    if (button.textContent != 'Backspace' && button.textContent != 'CapsLock' && button.textContent != 'Shift' &&
```

Comment:
You are using too many conditions at the if (). You shouldn't do it. It means than there are some issues in you logic